src/app/child-dev-project/attendance/attendance-details/attendance-details.component.spec.ts

```diff
@@ -40,7 +40,7 @@ describe('AttendanceDetailsComponent', () => {
         UiHelperModule, EntityModule],
       providers: [
         {provide: Database, useClass: MockDatabase},
-        {provide: MatDialogRef, useValue: {beforeClose: () => { return { subscribe: () => {}}; }}},
+        {provide: MatDialogRef, useValue: {beforeClosed: () => { return { subscribe: () => {}}; }}},
```

Comment:
why was this changed? according to the Angular API Reference it should be `beforeClose` (https://material.angular.io/components/dialog/api#MatDialogRef)

Reply:
Ooops, sorry, this must have been a typing error.